docs/content/docs/deployment/speculative_execution.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Speculative Execution
+weight: 5
+type: docs
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Speculative Execution
+Apache Flink supports speculative execution of batch jobs.
+
+This page describes the background of speculative execution and how to use it.
+
+## Background
+Speculative execution is a mechanism to mitigate job slowness which is caused by problematic nodes. 
+A problematic node may have hardware problems, accident I/O busy, or high CPU load. These problems may
+make the hosted tasks run much slower than tasks on other nodes, and affects the overall execution time 
+of a batch job.
+
+In such cases, speculative execution will start a new attempt of the slow task on nodes that are not 
+detected as problematic/slow. The new attempt processes the same input data and produces the same data 
```

Comment:
`problematic/slow` -> `problematic`. I think `problematic` is enough, which is same as the above description.